indexing-service/src/main/java/org/apache/druid/indexing/common/task/CompactionTask.java

```diff
@@ -562,6 +572,11 @@ private String createIndexTaskSpecId(int i)
     return StringUtils.format("%s_%d", getId(), i);
   }
 
+  private String getReportkey(String baseSequenceName, String currentKey)
```

Comment:
This method is not needed, please see the other comment regarding building the report key using index of item in list.